### DISCUSSION TITLE
Remove empty rows

### DISCUSSION THREAD

**Discussion by memark** [uiua-lang/uiua#244] [Learning] [answered]
I want to remove empty rows from a (boxed) array. Here are two ways I've come up with:

```uiua
{[1 2] [1] []}
▽>0≡⧻.

[⟦1 2⟧ ⟦1⟧]
```

```uiua
{[1 2] [1] []}
▽¬⊚≡⧻.

[⟦1 2⟧ ⟦1⟧]
```

Is there an easier/more idiomatic way?

**Comment by kaikalii** [comment, marked as answer]
`▽>0≡⧻.` seems like a sensible way to me.
